src/components/input-number/input-number.stories.ts

```diff
@@ -83,9 +83,7 @@ export const withSlottedAction = (): string => html`
     >
       <calcite-button slot="action">${text("action button text", "Go")}</calcite-button>
     </calcite-input-number>
-    <calcite-input-message
-      ${boolean("input-message-active", false)}
-      status="${select("input message status", ["idle", "valid", "invalid"], "idle")}"
+    <calcite-input-message hidden status="${select("input message status", ["idle", "valid", "invalid"], "idle")}"
```

Comment:
Same here, we need a way to toggle hidden in the story right?

Reply:
A few thoughts:

1. I'm not sure if we need the input message for the slotted action story/use case. This sort of applies to other types too.
2. I think it'd be more helpful to have a usage entry or simple example showing how input message can be toggled dynamically. Maybe this is a future validation story candidate.
3. Shouldn't any `withInputMessage` story always show it?